src/datastore.js

```diff
@@ -401,6 +401,7 @@ s  */
    * @param {String} dataStore The data store name
    * @param {String} wfsCapabilitiesUrl WFS capabilities URL
    * @param {String} namespaceUrl URL of the GeoServer namespace
+   * @param {Boolean} [useHttpConnectionPooling] use HTTP connection pooling for WFS connection, defaults to true
```

Comment:
accoriding to JSDocs this notation could be used as well - [link](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values)

```suggestion
   * @param {Boolean} [useHttpConnectionPooling=true] use HTTP connection pooling for WFS connection
```

Reply:
:+1: adapted by rebasing.